test/test.googleauth.ts

```diff
@@ -1229,6 +1230,13 @@ it('should accept credentials to get a client', async () => {
   assert.equal(client.email, 'hello@youarecool.com');
 });
 
+it('should allow passing scopes to get a client', async () => {
+  const scopes = ['http://examples.com/is/a/scope'];
```

Comment:
`scopes` can be an array or a string, right? Maybe it's worthwhile to have a test for the latter case too?

Reply:
good call :)